src/dmd/statement.d

```diff
@@ -449,6 +449,89 @@ extern (C++) abstract class Statement : ASTNode
     inout(CompoundDeclarationStatement) isCompoundDeclarationStatement() { return stmt == STMT.CompoundDeclaration ? cast(typeof(return))this : null; }
 }
 
+
+// TODO Make this function iterative if it turns out to be a bottleneck
```

Comment:
This comment is likely to be ignored forever.

Reply:
oh I've already addressed it just forgot to delete the comment.